generated/nidigitalpattern/nidigitalpattern.proto

```diff
@@ -41,7 +41,7 @@ service NiDigital {
   rpc ConfigureStartLabel(ConfigureStartLabelRequest) returns (ConfigureStartLabelResponse);
   rpc ConfigureTerminationMode(ConfigureTerminationModeRequest) returns (ConfigureTerminationModeResponse);
   rpc ConfigureTimeSetCompareEdgesStrobe(ConfigureTimeSetCompareEdgesStrobeRequest) returns (ConfigureTimeSetCompareEdgesStrobeResponse);
-  rpc ConfigureTimeSetCompareEdgesStrobe2x(ConfigureTimeSetCompareEdgesStrobe2xRequest) returns (ConfigureTimeSetCompareEdgesStrobe2xResponse);
+  rpc ConfigureTimeSetCompareEdgesStrobe2X(ConfigureTimeSetCompareEdgesStrobe2XRequest) returns (ConfigureTimeSetCompareEdgesStrobe2XResponse);
```

Comment:
Need to override the name in hapigen with 'grpc_name'.

Reply:
It's still showing up with 2X but I think you have it right in functions.py now. Can you rebuild locally and push those generated file changes up to update this?

Reply:
These diffs are still showing up here. Can you re-build locally and push up the changes to these files?

Reply:
@IPetersNI the latest still has a diff here. Contact me after lunch if you want me to help you figure it out locally.